src/Xrd/XrdPollE.hh

```diff
@@ -79,5 +79,6 @@ struct epoll_event *PollTab;
        int          PollMax;
        int          WaitFd;
 XrdSysSemaphore     WaitFdSem;
+XrdSysSemaphore     WaitFdSem2;
```

Comment:
One minor request for future readability --

For `WaitFdSem2` here (and `waitMtx`/`waitHelper`), can you document what data is being protected by the semaphoer/mutex and the locking invariants that should be followed?